src/main/java/com/example/demo/Carddata/Carddata.java

```diff
@@ -0,0 +1,36 @@
+package com.example.demo.Carddata;
+
+import jakarta.persistence.Entity;
+
+import jakarta.persistence.Table;
+
+@Entity
+@Table
+public class Carddata {
+    private Integer id;
+    private Double Card_num;
+
+    public Carddata() {
+    }
+
+    public Carddata(Integer id, Double card_num) {
+        this.id = id;
+        Card_num = card_num;
+    }
+
+    public Integer getId() {
```

Comment:
Remove this getter and setters, use Lombok annotations for automatic getter and setter generation

Reply:
done